float64 start_latitude
float64 start_longitude
float64 end_latitude
float64 end_longitude
---
bool result